src/patterns/page-not-found-pages/index.md.njk

```diff
@@ -55,22 +55,19 @@ Do not use:
 - informal or humourous words like oops
 - red text to warn people
 
-### Service mistake
+### If you do not know what caused the page not found
```

Comment:
I would suggest removing the other 2 examples, so I think we can remove this H3.

Reply:
Agreed.